DATA_OVERLAY.md

```diff
@@ -0,0 +1,58 @@
+
+The requirement for all values to be in the 0-1 range is to ensure values from different data sources can be handled the same way by the visualization component.
+
+When adding new data sources, it is important to inspect the range of the values. For example, if all of the values are within 0.01 and 0.02, only a small segment of the color spectrum will be used. Similarly, for uneven distribution, e.g. if 50% of the data is within 0.01 - 0.02 and 30% is within 0.98 - 0.99, the same visualization problem would be encountered.
+Ideally, the minimum value of the data source should be as close to 0 as possible, while the maximum value should be as close to 1 as possible, with as even distribution as possible between for the rest of the values.
+
+Data overlay also allows for two datasets to be selected at the same time. When this is the case, the following spectrum is used instead.
+
+<img width="300" alt="image" src="https://user-images.githubusercontent.com/423498/128190231-8b475022-109e-4905-8a01-5e862c397899.png">
+
+## Folder structure and naming conventions
+
+All data source files should be stored in the [data-files repo](https://github.com/MetabolicAtlas/data-files).
+
+Under each folder in [`/integrated-models`](https://github.com/MetabolicAtlas/data-files/tree/main/integrated-models), there should be a `dataOverlay` folder, for example: `data-files/integrated-models/Human-GEM/dataOverlay`. If not, it means the model currently does not have any data sources for data overlay and the `dataOverlay` folder should be added.
+
+Inside of the `dataOverlay` folder, there could be one or more folders, where the name of each folder should be the data type. Currently, the types `transcriptomics` and `metabolomics` are supported.
+
+Inside of a data type folder, there could be one or more data source files, such as `hpa.tsv`. There should also be an `index.tsv` file, which contains the metadata for all of the data sources in this folder. When adding a new data source, a new row should be added to the `index.tsv` file with the following values:
+
+- `filename`: name of the new data source file without the extension, for example: `hpa` for `hpa.tsv`
+- `name`: name of the data source to be displayed to the end user, for example: `The Human Protein Atlas`
+- `link`: website of the data source, for example: `https://www.proteinatlas.org`
+- `lastUpdated`: date for when the data source was last updated in `YYYY-MM-DD` format, for example: `2021-08-01`
+
+## Adding a new data source
+
+When incorporating new data sources, it should be done via a pull request in the [data-files repo](https://github.com/MetabolicAtlas/data-files). The above rules and conventions about data format as well as folder structure should be followed.
+
+If it's difficult to establish how to transform the values into the 0-1 scale, please add a note about this in the pull request description and set the pull request to draft mode.
```

Comment:
I think this line is not needed anymore but otherwise this PR looks good to be merged.
@e0 up to you if you would want to tag extra reviewers.

Reply:
Ah now I see what you mean: `in the pull request description` on line 58 refers to the pull request on line 56. I'd suggest joining the 2 paragraphs for extra clarity.

Somehow I understood that the transformation should be described in the current PR, which did not make much sense to me, as we established that this PR is not about transforming values.

Reply:
> Ah now I see what you mean: `in the pull request description` on line 58 refers to the pull request on line 56. I'd suggest joining the 2 paragraphs for extra clarity.
> 
> 
> 
> Somehow I understood that the transformation should be described in the current PR, which did not make much sense to me, as we established that this PR is not about transforming values.

Thanks for clarifying. I can see how it can be confusing in the context of this PR. This is in a standalone markdown file and there is a section header for these two paragraphs so I think it is already clear for the intended audience.